api/search_test.go

```diff
@@ -0,0 +1,155 @@
+package api
+
+import (
+	"testing"
+
+	"github.com/ONSdigital/dp-nlp-search-scrubber/api/mock"
+	"github.com/ONSdigital/dp-nlp-search-scrubber/models"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestGetAllMatchingIndustries(t *testing.T) {
+	// get a mock ScrubberDB with some industries
+	mockDB := mock.DB()
+
+	tests := []struct {
```

Comment:
An option here is to supply nothing and get nothing - probably sensible to catch that as a test.